pydy/viz/tests/test_shapes.py

```diff
@@ -57,30 +57,31 @@ def test_shape():
 def test_shape_geometry_with_expressions():
 
     shape = Shape()
-    shape.length = symbols('l')
+    shape.length = 16.0
```

Comment:
All of the changes in this file need to be reverted. These tests specifically test the symbolic functionality for the shape dimensions.